static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +111,28 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  createActionHelper: Function = (actionCreator: Function): (...args: any) => void  => {
+    const { dispatch } = this.props;
+    return (...args) => dispatch(actionCreator(...args));
+  };
```

Comment:
just wondering: why can't we just import dispatch where it's needed? why does it need to be passed down as a prop?